content/components/core/code/scss-functions.md

```diff
@@ -167,7 +167,9 @@ Add the outline to focus.
 
 ### AU-sronly
 
-Hide an element from the screen but not a screen reader.
+Hide an element from the screen but not a screen reader. This mixin should be used to provide additional information or context to a webpage or form that screenreader users would otherwise miss out on. Using this mixin the additional information will be picked up by a screenreader but will not be displayed in the browser. 
+
+An example of a use case for this class would be, providing more context to a 'read more' link on a webpage. Using this class, additional information about what the 'read more' link relates to could be hidden from the browser, but still picked up by a screenreader. This would provide a screenreader user with the required context to understand what information is being provided.   
```

Comment:
```suggestion
An example of a use case for this class could be to provide more context to a `read more` link on a webpage.

Using this class, additional information about what the `read more` link relates to could be hidden from the browser, but will still be picked up by a screen reader. This would provide a screen reader user with the required context to understand what information is being provided.
```